docs/src/main/asciidoc/_elytron/Elytron_Subsystem.adoc

```diff
@@ -369,8 +369,6 @@ information such as passwords for external services.
 |provider-loader |A definition for a provider loader.
 
 |security-domain |A security domain definition.
-
-|security-property |A definition of a security property to be set.
```

Comment:
Security properties were moved to be an attribute on the subsystem itself, we maybe should have a small section covering these attributes.

Reply:
Added security-property back